src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -8,7 +8,7 @@
  * Guarantees: immutable; name is valid as declared in {@link #isValidTagName(String)}
  */
 public class Tag {
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tag names should be alphanumeric and not empty";
```

Comment:
Did the behavior for Tag change? "t/" without any input is working for "add" and "edit" command.

Reply:
Confirmed that the behavior of Tag has not been changed